shutdownpipette.sh

```diff
@@ -43,17 +43,18 @@ if [ $# -gt 0 ]; then
     check_args "$@"
 fi
 
-sudo docker stop pipette
+#sudo docker stop pipette
+sudo kill -9 ryu-manager
```

Comment:
hmm, is there only ever one ryu-manager running?

Reply:
The startup script can have ryu-manger log its pid to a file (e.g):

--pid=/tmp/pipette.pid

However I imagine for non-debugging use pipette would be run in a container, so docker stop would be the way to go.

Reply:
i added code to collect PIDs for ryu and tcpdump so that we can shut both down cleanly